components/SettingsComponent.vue

```diff
@@ -54,76 +63,79 @@ const d = Object.fromEntries(Object.entries(data).filter(([k, v]) =>
 </style>
 
 <template>
- <template v-for="(v, k) in d">
-  <component :is="'h' +  level" :id="k" tabindex="-1">
-   <code>{{ k }}</code>
-   <a class="header-anchor" :href="'#' + k"></a>
-  </component>
+ <span class="settingsList">
+  <template v-for="(v, k) in d">
```

Comment:
Same as with both the `EventsComponent` and the `DoveadmComponent`:
- Use `<section>` instead of `<span>`, and
- use `<article>` instead of (inner) `<template>`.

And adapt the styles to say:
```css
.doveadmList article:first-of-type {
  /* ... */
}
.doveadmList article {
  /* ... */
}
```

Reply:
span changed to section

However, like https://github.com/dovecot/documentation/pull/1000 , template is the correct usage here, since we need to loop between the parent container and the h3 sections, and we don't want anything output to the page.

Reply:
I replied to your reply, let's keep this open until a solution can be decided on.